src/lib/djerba/plugins/sample/render.py

```diff
@@ -0,0 +1,66 @@
+"""collection of functions for rendering Djerba content in Mako"""
+
+import djerba.plugins.sample.constants as constants
+import re
+from markdown import markdown
+from time import strftime
+from string import Template
+from djerba.util.image_to_base64 import converter
+
+class html_builder:
+
+    TR_START = '<tr style="text-align:left;">'
+    TR_END = '</tr>'
+
+    def section_cells_begin(self, section_title, main_or_supp):
```

Comment:
I've put "section cells begin/end" methods in the `util.html` package: https://github.com/oicr-gsi/djerba/blob/GCGI-806_v1.0.0-dev/src/lib/djerba/util/html.py

Could you use the `util` methods instead of duplicating them here please?

Reply:
Fixed!